Implementation and models used in the paper

pawni opened the pull request:
This is code for the models used in the DLTK paper.

mrajchl commented:
Sweet, detailed implementation. Love it. 
Issues: 
- When running preprocessing.py the subfolders of the output_path (i.e. img/ and label/) are not created and result in write errors.
- All models train, however require at LEAST 16 GB mem cards (i.e. config_asym_unet_balce.json). This could be stated in the README.md as these are not that available.
- tqdm in preprocessing.py is an extra dependency that is not provided with a dltk install.
- Travis complains

pawni commented:
I fixed the creation of the subfolders, deleted tqdm and changed the batch size to 4. I also added a note about the batch size to the readme.

Travis complains because of other files in this repo.

mrajchl commented:
Looks good.
